core/appeal/service.go

```diff
@@ -797,6 +797,13 @@ func getApprovalNotifications(appeal *domain.Appeal) []domain.Notification {
 						"requestor":     appeal.CreatedBy,
 						"appeal_id":     appeal.ID,
 						"account_id":    appeal.AccountID,
+						"account_type":  appeal.AccountType,
+						"provider_type": appeal.Resource.ProviderType,
+						"resource_type": appeal.Resource.Type,
+						"created_at":    appeal.CreatedAt,
+						"approval_step": approval.Name,
+						"actor":         approver,
+						"details":       appeal.Details,
```

Comment:
how about adding `appeal` object, `approval` object, and `approver` string and allow user to use expression ta access them? while the existing fields are kept for backward compatibility

Reply:
This change is needed if we are going with the expression change.
But as checked , facing some error while using expression for templates, Will take it in a separate Pr.

Reply:
@utsav14nov please create an issue for that